src/components/input/styles.ts

```diff
@@ -146,10 +150,12 @@ export const Input = styled.input<IInput>`
 
 export const WrapperReset = styled.div`
   position: absolute;
-  top: 12px;
+  top: 50%;
   right: 12px;
+  display: flex;
   cursor: pointer;
   transition: 0.33s;
+  transform: translateY(-50%);
 
   svg {
     width: 24px;
```

Comment:
And I guess You forgot to export your component from storybook
components/index.ts